creator-node/src/models/file.js

```diff
@@ -102,9 +102,16 @@ module.exports = (sequelize, DataTypes) => {
     // sequelize does not support composite foreign keys
   }
 
-  // TODO - why is this not externally accessible?
   File.TrackTypes = ['track', 'copy320']
   File.NonTrackTypes = ['dir', 'image', 'metadata']
 
+  File.Types = {
```

Comment:
nice, this is better. can you change lines 105 and 106 accordingly? File.TrackTypes = [File.Types.track, File.Types.copy320] etc